src/CRM/CivixBundle/Command/InitCommand.php

```diff
@@ -103,6 +107,30 @@ protected function execute(InputInterface $input, OutputInterface $output) {
       $output->writeln('<error>Unrecognized license (' . $ctx['license'] . ')</error>');
       return;
     }
+
+    if ($ctx['fullName'] !== $ctx['mainFile']) {
+      $output->writeln("");
+      $output->writeln("<info>ALERT: The requested command requires split-naming.</info>");
```

Comment:
Overall looks good. A little bit of a mismatch now where it says "ALERT" but has green info styling. But not a blocker.

Reply:
Tweaked:

<img width="369" alt="Screen Shot 2022-02-14 at 2 29 26 PM" src="https://user-images.githubusercontent.com/1336047/153957414-fe359673-6bc5-4ab7-80bd-685950ab3b74.png">

Reply:
Looks good 👍